src/clang.rs

```diff
@@ -415,9 +415,14 @@ impl Cursor {
     ///
     /// Returns -1 if the cursor's referent is not a function/method call or
     /// declaration.
-    pub fn num_args(&self) -> i32 {
+    pub fn num_args(&self) -> Option<u32> {
```

Comment:
I guess `Result<T, ()>` let's us use `try!()`, so that's a slight advantage over `Option` even though the error variant is empty.